README.md

```diff
@@ -1,8 +1,13 @@
-# StochSS: The Stochastic Simulation Service
+# StochSS: Software as a service for simulation-driven investigations of stochastic biochemical models
 
-StochSS is a platform for simulating biochemical systems supporting both continuous ODE and discrete stochastic simulations with [GillesPy2](https://github.com/GillesPy2/GillesPy2). 
+StochSS provides advanced tools for simulating and analyzing biochemical systems as Software as a Service. StochSS supports a wide range of model types ranging from continuous ODEs to discrete (spatial) stochastic simulations as well as advanced model analysis workflows. Try out StochSS here: https://app.stochss.org .
 
-At the moment StochSS development on Windows is not supported. You can try using [Make for Windows](http://gnuwin32.sourceforge.net/packages/make.htm), but this is untested!
+## Why use StochSS
+StochSS provides both an easy-to-use UI for constructing biochemical models as well as intuitive UIs for the most common model analysis tools such as visualization of results, parameter sweeps and parameter inference. StochSS is built on top of Jupyter Hub, and this lets you seemlessly switch between UI representations and Notebooks exposing the full backend API. In this way a user can benefit from the best of graphical representations with a clear upgrade path to modeling as code. For this reason, StochSS is also uniquely suited for the computational biologist starting out with mathematical modeling of biochemical systems. 
+
+StochSS is a sotware project involving several leading research groups in computational systems biology. We welcome collaborators to help expand the capabilities of StochSS. Read more about the project at www.stochss.org .   
+
+*StochSS is intended to be used as Software as a Service. The instructions that follows help you set your own local development environment or deploy your own instance as SaaS. If you intend to mainly use StochSS for modeling, simulation and model analysis, consider using the hosted deployment at http://app.stochss.org which you can access without any installation. 
 
 ## Requirements
```

Comment:
Need to make it clear that everything below here is for experts only.  "If you want to run your own server..."